### ISSUE TITLE
First describe_certificate call does not return DomainValidationOptions

### ISSUE THREAD

**Issue by mseiwald** [dflook/cloudformation-dns-certificate#6]
Since yesterday we are running into the issue that the Lambda fails with the following error:

```
Traceback (most recent call last):
File "/var/task/index.py", line 74, in handler
j()
File "/var/task/index.py", line 57, in j
for F in K[Z]:
KeyError: 'DomainValidationOptions'

```

Apparently this is because the first call to `describe_certificate` [here](https://github.com/dflook/cloudformation-dns-certificate/blob/master/src/troposphere_dns_certificate/certificate.py#L211) does not return `DomainValidationOptions`. After adding a `sleep(10)` directly in front of the `describe_certificate` it starts working again.

**Comment by dflook** [comment]
Thanks for creating an issue, I'll look at this today.

**Comment by dflook** [comment]
What region are you creating the certificate in? I can't seem to reproduce this in eu-north-1.

**Comment by mseiwald** [comment]
eu-west-1

**Comment by dflook** [comment]
This stopped working in eu-west-1 and possibly some other regions due to a change in ACM API behaviour. It's fixed in 1.7.1 and verified to work in all regions.
